.claude/skills/divio-docs/SKILL.md

```
@@ -0,0 +1,279 @@
+
+- **Returns:** `{return type}`
+- **Description:** {What it does}
+
+## Examples
+
+```{language}
+{code example}
+```
+````
+
+````
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)